scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
If you do `from website import settings`, it consolidates settings from defaults.py and local.py in the settings/__init__.py file.  local.py is just what is on your machine - and not added to source control. So the only place to put this is in `defaults`.  Local will override website/settings/defaults.py, but it is only on your machine.

I was mainly extending this PR https://github.com/CenterForOpenScience/osf.io/pull/4879/files